### DISCUSSION TITLE
The Spreadsheet Note Type Doesn't work on iPad 12.9 iOS 16.6.  As it's provided by an external developer, the discussion seems to have stopped. Maybe Standard Notes should withdraw it from their list of features.

### DISCUSSION THREAD

**Discussion by Pendyploo** [standardnotes/forum#3256] [Suggestions]
After you create a Spreadsheet note type, the Spreadsheet frequently does not respond at all. The Keyboard does not appear. Cells cannot be accessed.  Menu icons seem to be active, but don't do anything. Very rarely, sometimes you're able to enter data in one cell, but then you can't edit that data, and can't access other cells.  You are unable to alter column widths. if you add a new Sheet to the Spreadsheet you can enter data in the top RHS cell. The only way to move to another cell is to use Tab or Shift-Tab.  Haven't found a way to move to cells upwards or downwards. You can't enter formulas.  iPad oS 16.6.  iPad 12.9

**Comment by Pendyploo** [comment]
Three months later, I haven’t been able to trace a response or query.  Maybe Standard Notes should admit that the Spreadsheet note type doesn’t work, and withdraw it.  Hope im wrong.  Anyone out there have anything to add?

**Reply by UncleRobin** [reply]
Well that explains a few things. I used or used to use the spreadsheet type note a lot.  I am not sure when I discovered spreadsheets types didn't work on iOS, but it has been a while back.  It would have been nice if support would have mentioned the code comes from an external source and that was the issue vs the blank stare in the headlights replies.
